rpm/boss-launcher-webhook.spec

```diff
@@ -29,37 +29,55 @@ Summary: VCS webhook handler
 Webhook handler for gitlab, github and bitbucket that receives data as a POST callback and launches a ruote process
 
 %package -n obs-service-tar-git
-Requires: git, obs-source_service
+Requires: git
+Requires: obs-source_service
 Summary: OBS source service to generate sources from git
 %description -n obs-service-tar-git
 This package provides the service to generate source from git inside an OBS source service
 
 %package -n obs-service-webhook
-Requires: obs-source_service, python-argparse, python-requests
+Requires: obs-source_service
+Requires: python-argparse
+Requires: python-requests
 Summary: OBS source service to manage webhooks
 %description -n obs-service-webhook
 This package provides the service to update webhooks from OBS. It ensures that only users who have access to a package can update the webhook for that package.
 
 %package -n boss-participant-trigger_service
-Requires: python-boss-skynet >= 0.6.0, boss-standard-workflow-common, python-lxml, python-yaml, python-buildservice >= 0.5.3
+Requires(post): boss-standard-workflow-common
+Requires(post): python-boss-skynet >= 0.6.0
+Requires: osc
+Requires: python-boss-common >= 0.27.0
```

Comment:
maybe use a %{participant_requires} macro to DRY for the common 3? Just a nitpick :)